pennylane/ops/op_math/pow.py

```diff
@@ -37,6 +37,59 @@
 _superscript = str.maketrans("0123456789.+-", "⁰¹²³⁴⁵⁶⁷⁸⁹⋅⁺⁻")
 
 
+def pow(base, z=1, lazy=True, do_queue=True, id=None):
+    """Raise an Operator to a power.
+
+    Args:
+        base (~.operation.Operator): the operator to be raised to a power
+        z=1 (float): the exponent
+
+    Keyword Args:
+        lazy=True (bool): In lazy mode, all operations are wrapped in a ``Pow`` class
+            and handled later. If ``lazy=False``, operation-specific simplifications are first attempted.
+
+    Returns:
+        Operator
+
+    .. seealso:: :class:`~.Pow`, :meth:`~.Operator.pow`.
+
+    **Example**
+
+    >>> qml.pow(qml.PauliX(0), 0.5)
+    PauliX(wires=[0])**0.5
+    >>> qml.pow(qml.PauliX(0), 0.5, lazy=False)
+    SX(wires=[0])
+    >>> qml.pow(qml.PauliX(0), 0.1, lazy=False)
+    PauliX(wires=[0])**0.1
+    >>> qml.pow(qml.PauliX(0), 2, lazy=False)
+    Identity(wires=[0])
+
+    Lazy behavior can also be accessed via ``op ** z``.
+
+    """
+    if lazy:
+        return Pow(base, z, do_queue=do_queue, id=id)
+    try:
+        pow_ops = base.pow(z)
+    except PowUndefinedError:
+        return Pow(base, z)
+
+    num_ops = len(pow_ops)
+    if num_ops == 0:
+        # needs to be identity (not prod of identities) so device knows to skip
```

Comment:
Then we should change all the product of identities used in the simplify pipeline...

Reply:
Yep, but I don't think we can make it for this release 😢